test/fixtures/products.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
IF we will use Rspec you do not need fixtures, you can use factories. See FactoryBot gem.